File: core/src/com/bombbird/terminalcontrol/utilities/ToastManager.java
package com.bombbird.terminalcontrol.utilities;

import com.badlogic.gdx.utils.GdxRuntimeException;

public interface ToastManager {
    default void saveFail(GdxRuntimeException e) {
        //No default implementation
    }

    default void readStorageFail() {
        //No default implementation
    }

    default void jsonParseFail() {
        //No default implementation
    }
}

File: core/src/com/bombbird/terminalcontrol/utilities/ToastManager.kt
package com.bombbird.terminalcontrol.utilities

import com.badlogic.gdx.utils.GdxRuntimeException

interface ToastManager {
    fun saveFail(e: GdxRuntimeException) {
        //No default implementation
    }

    fun readStorageFail() {
        //No default implementation
    }

    fun jsonParseFail() {
        //No default implementation
    }
}